index.test.js

```diff
@@ -487,7 +488,7 @@ describe('Deploy to ECS', () => {
 
         expect(mockEcsUpdateService).toHaveBeenCalledTimes(0);
         expect(mockEcsWaiter).toHaveBeenCalledTimes(0);
-    });
+    }); 
```

Comment:
could you remove the trailing whitespace?

Reply:
Done, thank you